JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -61,10 +61,15 @@ private static Connection connectToDB(String host, String port, String dbName, S
         Properties jdbcOptions = new Properties();
         jdbcOptions.put("oauthaccesstoken", accessToken);
         jdbcOptions.put("oauthrefreshtoken", refreshToken);
-        jdbcOptions.put("oauthtokenurl", tokenUrl);
-        jdbcOptions.put("oauthclientid", clientId);
-        jdbcOptions.put("oauthclientsecret", clientSecret);
-	jdbcOptions.put("oauthscope", scope);
+
+	// Put these options into static json config
+	String jsonConfig = "{\"oauthtokenurl\" : \"" + tokenUrl + "\", " +  
+	"\"oauthclientid\" : \"" + clientId + "\", " + 
+	"\"oauthclientsecret\" : \"" + clientSecret + "\", " +
+	"\"oauthscope\" : \"" + scope + "\"" +
+	"}";
```

Comment:
This makes it harder to read than before.  Too bad Python-like multi-line strings are only available in Java 15+.

Do we have to make this change for these examples (i.e. do we recommended always using the JSON for some of the OAuth params)?

Reply:
Yes, the change in functionality is to use json configuration for static configuration, and only maintain the parameters for that configuration for backwards compat.